### PR TITLE
Add Inference Windows CUDA documentation

### PR DESCRIPTION
**Pull request by capjamesg**
# Description

This PR adds documentation that shows how to install Inference on Windows with a CUDA GPU.

## Type of change

Documentation update.

## How has this change been tested, please provide a testcase or example of how you tested the change?

You can open the documentation page to make sure it is properly formatted.

## Any specific deployment considerations

N/A

## Docs

N/A

### CODE REVIEW

**Review by PawelPeczek-Roboflow — COMMENTED** [review]
I do not have ability to verify commands due to lack of Windows machine - was that checked e2e? If so - docs looks legit

**Comment by capjamesg** [comment]
This is what the FE team sends to customers. @sberan and @chandlersupple can confirm. I have just tagged you both as reviewers.

**Comment by sberan** [comment]
> This is what the FE team sends to customers. @sberan and @chandlersupple can confirm. I have just tagged you both as reviewers.

FYI I haven’t sent this to customers, it’s been internal only.

**Review by sberan — CHANGES_REQUESTED** [review]
@capjamesg thanks for getting these formatted! Could you add an important blurb at the very top encouraging users to use docker as it is the preferred method of installing and will skip all this headache? I worry that people will think this is the recommended approach on Windows.

File: mkdocs.yml
Comment (sberan) [inline mkdocs.yml:98]:
```suggestion
          - Install “bare metal” Inference GPU on Windows: quickstart/inference_gpu_windows.md
```

**Comment by PawelPeczek-Roboflow** [comment]
@sberan good point!

**Review by grzegorz-roboflow — APPROVED** [review]

**Comment by capjamesg** [comment]
@sberan I have made the requested change. Please review.

**Review by sberan — APPROVED** [review]
LGTM!

**Comment by capjamesg** [comment]
We need a review from a code owner to merge cc @PawelPeczek-Roboflow.

**Review by PawelPeczek-Roboflow — APPROVED** [review]

**Pull request merged**
